src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2443,12 +2443,6 @@ private uint getThreadTLSIndex(ref void* ppIndirection)
         private void* getInlinedCallFrameVptr(ref void* ppIndirection)
         { throw new NotImplementedException("getInlinedCallFrameVptr"); }
 
-        private int* getAddrOfCaptureThreadGlobal(ref void* ppIndirection)
```

Comment:
What about CoreRT? Is this something that we will need to handle during that merge?

Reply:
Yes, it is the usual procedure.